### PR TITLE
Move the disk metrics that are not already covered by upstream

### PR DESCRIPTION
**Pull request by adamhf**

### CODE REVIEW

File: rules/rules.libsonnet [hunk 1]
```diff
@@ -15,6 +15,17 @@
           },
         ],
       },
+      {
+        name: 'mintel-disk.rules',
+        rules: [
+          {
+            record: 'mintel:pvc:inodes_free:percentage',
+            expr: |||
+              (kubelet_volume_stats_inodes_free / kubelet_volume_stats_inodes) * 100
```
Comment (primeroz) [inline rules/rules.libsonnet]: please add the "Kubelet Selector" using the variable, similar the other rule in this file

File: alerts/alerts.libsonnet [hunk 2]
```diff
@@ -24,6 +24,43 @@
...
+        name: 'mintel-disks',
+        rules: [
+          {
+            alert: 'KubePersistentVolumeInodeUsageCritical',
+            expr: |||
+              mintel:pvc:inodes_free:percentage <= 5
+            |||,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
```
Comment (primeroz) [inline alerts/alerts.libsonnet]: should probably be critical

Comment (primeroz) [reply alerts/alerts.libsonnet]: also don't need the context label, it was a legacy thing

File: alerts/alerts.libsonnet [hunk 3]
```diff
@@ -24,6 +24,43 @@
...
+          },
+          {
+            alert: 'KubePersistentVolumeInodePredictedUsageCritical',
+            expr: |||
+              predict_linear(mintel:pvc:inodes_free:percentage[4h], 4 * 24 * 3600) <= 0
+            |||,
+            'for': '10m',
+            labels: {
+              severity: 'critical',
+              context: 'cluster',
```
Comment (primeroz) [inline alerts/alerts.libsonnet]: no need of context label

File: alerts/alerts.libsonnet [hunk 4]
```diff
@@ -24,6 +24,43 @@
           },
         ],
       },
+      {
+        name: 'mintel-disks',
```
Comment (primeroz) [inline alerts/alerts.libsonnet]: what about the Inodes rule for standard , non volumes, disks and the usage for kubelet volumes ?

File: rules/rules.libsonnet (hunk 1 above)
Comment (adamhf) [reply rules/rules.libsonnet]: Done

File: alerts/alerts.libsonnet (hunk 2 above)
Comment (adamhf) [reply alerts/alerts.libsonnet]: Done

File: alerts/alerts.libsonnet (hunk 3 above)
Comment (adamhf) [reply alerts/alerts.libsonnet]: Done

File: alerts/alerts.libsonnet [hunk 5]
```diff
@@ -24,6 +24,41 @@
...
+            expr: |||
+              predict_linear(mintel:pvc:inodes_free:percentage[4h], 4 * 24 * 3600) <= 0
+            |||,
+            'for': '10m',
+            labels: {
+              severity: 'critical',
+            },
+            annotations: {
+              description: 'The persistent volume {{ $labels.persistentvolumeclaim }} in namespsace\n                {{ $labels.exported_namespace }} is predicted to use all its inodes within the\n                next 4 days',
+              runbook_url: 'https://gitlab.com/mintel/satoshi/monitoring/runbooks/blob/master/satoshi/platform/KubePersistentVolumeInodePredictedUsageCritical.md',
```
Comment (nabadger) [inline alerts/alerts.libsonnet]: We can fix this reference in a different MR (and batch-update the new runbook urls)

**Pull request merged**